src/webtransport.js

```diff
@@ -606,9 +858,13 @@ export class Http3Server extends Http3WebTransport {
     this.stopped = true
   }
 
+  /**
+   * @param {string} path
+   * @returns {ReadableStream<WebTransportSession>}
+   */
   sessionStream(path) {
     if (path in this.sessionStreams) {
-      return this.sessionsStreams[path]
+      return this.sessionStreams[path]
```

Comment:
This appears to be a typo, it would likely explode if invoked at runtime.

Reply:
I think it is a typo. Interesting that it did not cause harm so far.